pallets/identity/src/lib.rs

```diff
@@ -1773,10 +1786,23 @@ impl<T: Trait> Module<T> {
         issuer: IdentityId,
         scope: Option<Scope>,
     ) -> DispatchResult {
+        // Ensure the target is the issuer of the claim.
         ensure!(
             target == issuer,
             Error::<T>::ConfidentialScopeClaimNotAllowed
         );
+        let (pk, sk) = Self::get_claim_keys(target, claim_type, issuer, scope.clone());
+        if let IdentityClaim {
+            claim: Claim::InvestorUniqueness(_, scope_id, _),
+            ..
+        } = <Claims>::get(&pk, &sk)
+        {
+            // Ensure that the target has balance at scope = 0.
+            ensure!(
+                T::AssetSubTraitTarget::balance_of_at_scope(&scope_id, &target) == Zero::zero(),
```

Comment:
In balances, we have the **Existential deposit** concept, which is the minimum balance required to create or keep an account open. 
I am wondering if we need same concept here and we should use something like:
```Rust
balance_of_at_scope(&scope_id, &target) < Balances::ExistentialDeposit::get()
```
**NOTE**: If it is not **strictly** necessary, we could _avoid_ that because I guess it would introduce a new dependency between Identity & Balances which could create a circle dep. or a refactor in their traits or subtraits.

Reply:
Yeah, substrate offers the Existential deposit concept but we anyway set it to 0 so it is the same. And yes I don't want to battle with circular dependecies 😉